UpsertCgnStatus/index.ts

```diff
@@ -36,7 +36,7 @@ const app = express();
 secureExpressApp(app);
 
 // Add express route
-app.delete("/api/v1/cgn/:fiscalcode", RevokeCgn(userCgnModel));
+app.post("/api/v1/cgn/status/:fiscalcode", UpsertCgnStatus(userCgnModel));

```

Comment:
```suggestion
app.post("/api/v1/cgn/:fiscalcode/status", UpsertCgnStatus(userCgnModel));

```

Reply:
Fixed in [0f2f273](https://github.com/pagopa/io-functions-cgn/pull/12/commits/0f2f273bd086ff2a263a5e50104cb22cbcb96cc4)